Druid/DRCogitCodeGenerator.class.st

```diff
@@ -1189,6 +1189,14 @@ DRCogitCodeGenerator >> visitNoOverflow: aDRNoOverflow inBranch: aDRBranchIfCond
 	^ aDRBranchIfCondition 
 ]
 
+{ #category : 'visiting' }
+DRCogitCodeGenerator >> visitNoTestMask: aDRNoTestMask inBranch: aDRBranchIfCondition [
+	
+	self visitTestMask: nil inBranch: aDRBranchIfCondition.
+	
+	^ aDRBranchIfCondition reversed
+]
```

Comment:
Why a `nil` here?

Reply:
Because that method's parameter is a `DRTestMask`, and I don't have one here (I have a `DRNoTestMask`). I thought of passing that, but I think it'd be more error-prone in the future, since someone may assume it's always a `DRTestMask`
That argument isn't used anyways, so it doesn't make much of a difference

Reply:
Ookkkkk maybe would be nice to have a comment :)